incubator/sparkoperator/values.yaml

```diff
@@ -0,0 +1,6 @@
+operatorImageName: lightbend/sparkoperator
+operatorVersion: 2.3.1
+operatorNamespace: spark-operator
```

Comment:
You mention in a doc `sparkoperator`, but default is `spark-operator`. Change to `sparkoperator` ?

Reply:
Good catch! I updated the doc instead because of the build issue I mentioned in an earlier comment.